src/index.ts

```diff
@@ -2353,20 +2356,26 @@ function getLatestContractVersion({
 			})
 
 		config.verbose && console.log('Contract Versions found:', versions)
+		let _versions = versions
```

Comment:
Since the version sorting code is a bit complex, i think it'd benefit of a little extra explainer of what its sorting for 

e.g. "sorts contract versioons by descending order. Example: [v4.3, v6.7, v2.1] --> [....]"

i'd add this comment beneath "// Filter keys starting with "v" or "Bv" based on type"